.travis.yml

```diff
@@ -41,3 +57,7 @@ script:
       cd ../test/unit_tests/cuda;
       make;
     fi
+
+after_script:
+  - sleep 30s
```

Comment:
Definitely this is not 100% working and correct solution and it wouldn't work always. I have multiple suggestions:
1. Put `valgrind` code execution to a script. It might be (no guarantee) that file would be saved before exiting the script as outstanding task. After the script we could easily display results
2. If above suggestion doesn't work try to use loop inside script, something like this (pseudo-code):
```bash
while( file is not created or file cannot be opened with write permission) then
    sleep 1 second
end
```
3. If non of solution work try to search for the viewers/text editors which could read file until writing to it would be finished